logic/minecraft/onesix/OneSixRead.cpp

```diff
@@ -114,10 +115,11 @@ LibraryPtr readRawLibrary(const QJsonObject &libObj, const QString &filename)
 
 LibraryPtr OneSixFormat::readRawLibraryPlus(const QJsonObject &libObj, const QString &filename)
 {
+	using namespace Json;
 	LibraryPtr lib = readRawLibrary(libObj, filename);
 	if (libObj.contains("insert"))
 	{
-		QJsonValue insertVal = libObj.value("insert");
+		QJsonValue insertVal = ensureJsonValue(libObj.value("insert"), "library insert rule");
```

Comment:
I see your point, though I only half agree with it. I'll decide if I'll change this later.